.travis.yml

```diff
@@ -1,10 +1,14 @@
 os: linux
 dist: xenial
 language: python
-python: 3.8
+python: 3.8.5
```

Comment:
It's annoying that Travis doesn't update the `3.8` alias when a new patch comes out.  Any reason not to specify `3.8.6` here?

Reply:
Doh!  Good point.  Done.